@xen-orchestra/proxy/src/app/mixins/backups/_PoolMetadataBackup.js

```diff
@@ -5,7 +5,7 @@ import { asyncMap } from '../../../_asyncMap'
 import { forkStreamUnpipe } from './_forkStreamUnpipe'
 import { Task } from './_Task'
 
-const DIR_XO_POOL_METADATA_BACKUPS = 'xo-pool-metadata-backups'
+export const DIR_XO_POOL_METADATA_BACKUPS = 'xo-pool-metadata-backups'
```

Comment:
IMHO, it should be the other way around, it should be defined in `RemoteAdapter`.

Maybe, at the end, it will not even be necessary in this file :slightly_smiling_face: